Readme.md

```diff
@@ -59,6 +59,9 @@ sent to the client (optional)
 * `addHook`: If `false`, this plugin will not register `onRequest` hook automatically,
    instead it provide two decorations `fastify.verifyBearerAuth` and
    `fastify.verifyBearerAuthFactory` for you.
+* `verifyErrorLogLevel`: string specifying the log level when verify error (optional).
+   The value must be vaild log level support by fastify, otherwise exception will be throwed
+   when register the plugin. default value is `error`.
```

Comment:
```suggestion
* `verifyErrorLogLevel`: an optional string specifying the log level when there is a verification error.
   It must be a valid log level supported by fastify, otherwise an exception will be thrown
   when registering the plugin. By default, this option is set to `error`.
```